Dockerfile

```diff
@@ -33,18 +33,18 @@ RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y $BU
 
 FROM $DISTRIB_NAME${DISTRIB_MAJOR} AS tomcat8
 ENV TOMCAT_MAJOR 8
-ENV TOMCAT_VERSION 8.5.72
-ENV TOMCAT_SHA512 41d7eb83120f210d238d8653d729bfb2be32f3666e6c04e73607c05f066c4136b0719f8107cf66673333548c82dc5b9c0357e91fc0ac845e64f055b598f27049
+ENV TOMCAT_VERSION 8.5.76
+ENV TOMCAT_SHA512 7b84a311b2ba3b6c92eea5739275b45686ed893bc000c16ead0a3cfe7c166b12d42485e9eb9c40fe279d207a293c4de65db3107602794f2b8e6071bc4d2b53ed
 
 FROM $DISTRIB_NAME${DISTRIB_MAJOR} AS tomcat9
 ENV TOMCAT_MAJOR 9
-ENV TOMCAT_VERSION 9.0.54
-ENV TOMCAT_SHA512 83430f24d42186ce2ff51eeef2f7a5517048f37d9050c45cac1e3dba8926d61a1f7f5aba122a34a11ac1dbdd3c1f6d98671841047df139394d43751263de57c3
+ENV TOMCAT_VERSION 9.0.59
+ENV TOMCAT_SHA512 74902b522abda04afb2be24d7410d4d93966d20fd07dde8f03bb281cdc714866f648babe1ff1ae85d663774779235f1cb9d701d5ce8884052f1f5efca7b62c68
 
 FROM $DISTRIB_NAME${DISTRIB_MAJOR} AS tomcat10
 ENV TOMCAT_MAJOR 10
-ENV TOMCAT_VERSION 10.0.12
-ENV TOMCAT_SHA512 e084fc0cc243c0a9ac7de85ffd4b96d00b40b5493ed7ef276d91373fe8036bc953406cd3c48db6b5ae116f2af162fd1bfb13ecdddf5d64523fdd69a9463de8a3
+ENV TOMCAT_VERSION 10.0.17
+ENV TOMCAT_SHA512 c9a8894ddedb9a4966abb8698a029c00dda2603acb798bbf11a99cf585c2e5db967bee006689ea148b819fa11d1d8fe1e799ceb09243e70b1310ca18ada1f17f
```

Comment:
we are not really building tomcat 10 anymore (@alxgomz knows why), I would suggest to remove this section to avoid thinking that is still relevant

Reply:
because none of our products supports it

Reply:
it was originally requested by product than later retired as it does require major product changes so it makes sense to remove it, I will do a separate PR though

Reply:
fixed in #72